### DISCUSSION TITLE
compatibility with php 8.2.4 version too

### DISCUSSION THREAD

**Discussion by meghakundu** [firefly-iii/firefly-iii#9205] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

If at all it would have been compatible with php version8.2.4 too, it would have been easier to view other features.
![image](https://github.com/user-attachments/assets/ede881f8-6c09-46d0-9e55-120a844ca90c)

### Solution

Would have opened http://127.0.0.1:8000

**Comment by JC5** [comment]
Hi, thanks for opening an issue.

Firefly III requires PHP 8.3 or greater.
